client/src/components/Signin.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from "react";
+import { UikInput, UikButton } from "../@uik";
+import "../@uik/styles.css";
+import axios from "axios";
+import styled from "styled-components";
+
+const Container = styled.div`
+  display: flex;
+  width: 400px;
+  margin: 0 100px;
+  flex-direction: column;
+`;
+
+class Signin extends Component {
+  state = {
+    username: "",
+    password: ""
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+
+    const endpoint = "http://localhost:3000/api/login";
+    axios
+      .post(endpoint, this.state)
+      .then(res => {
+        console.log(res.data);
+        localStorage.setItem("jwt", res.data.token);
+      })
+      .catch(err => {
+        console.log("ERROR");
```

Comment:
A console.log is not bad on line 31, but it does not let the user visually know that the login failed. You can set a boolean on state that is true when the login fails which dynamically renders different data such as an error message in JSX that is visible to the user.